custom-connector-sdk/src/main/java/com/amazonaws/appflow/custom/connector/model/ErrorCode.java

```diff
@@ -76,5 +76,10 @@
      * Unknown Error from the Application. Use this ErrorCode only when you are not able to use the
      * other specific error codes.
      */
-    UnknownError
+    UnknownError,
+
+    /**
+     * Specifies that the connector is failed to write all the records to the Application.
```

Comment:
This is to indicate that the connector encountered failure, for some of the records, while writing to the application.

Reply:
done